src/main/java/teammates/ui/webapi/action/ArchiveCourseAction.java

```diff
@@ -26,7 +27,10 @@ public void checkSpecificAccessControl() {
     @Override
     public ActionResult execute() {
         String idOfCourseToArchive = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
-        String archiveStatus = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ARCHIVE_STATUS);
+
+        CourseArchiveRequest courseArchiveRequest = getAndValidateRequestBody(CourseArchiveRequest.class);
```

Comment:
`CourseArchiveRequest` -> `ArchiveCourseRequest`?

Reply:
Sure, but this seems to break convention? All other requests are named <Object> <Action> <Request> i.e .CourseBasicRequest, CourseSaveRequest etc.

Reply:
Yup. Notice that. Sorry.